### PR TITLE
Add support for Kali linux

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request pulls in upstream changes from [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) and adds support for Kali Linux.

## 💭 Motivation and Context

Support for Kali Linux is required for the Kali AMI we are building for the COOL.

## 🧪 Testing

All pre-commit hooks and molecule tests pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
🐉

File: meta/main.yml [hunk 1]
```diff
@@ -8,11 +8,14 @@ galaxy_info:
   platforms:
     - name: Amazon
       versions:
-        - 2018.03
+        - 2
```
Comment (felddy) [inline meta/main.yml:11]: How does this work?  It's not in the list: https://aws.amazon.com/amazon-linux-ami/

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -6,8 +6,8 @@ driver:
 lint:
   name: yamllint
 platforms:
-  - name: amazon2018.03
-    image: amazonlinux:2018.03
+  - name: amazonlinux2
```
Comment (felddy) [inline molecule/default/molecule.yml:9]: Same question.

**Review by felddy — APPROVED** [review]
"⭐️⭐️⭐️⭐️SHOCKING AND UNFORGETTABLE."

File: meta/main.yml (hunk 1 above)
Comment (jsf9k) [reply meta/main.yml:11]: These are Docker images, not AMIs.  See [here](https://hub.docker.com/_/amazonlinux/?tab=description) for more details.

File: molecule/default/molecule.yml (hunk 2 above)
Comment (jsf9k) [reply molecule/default/molecule.yml:9]: These are Docker images, not AMIs.  See [here](https://hub.docker.com/_/amazonlinux/?tab=description) for more details.

**Pull request merged**